incubator/hnc/cloudbuild.yaml

```diff
@@ -57,6 +57,6 @@ steps:
   - 'https://uploads.github.com/repos/kubernetes-sigs/multi-tenancy/releases/$_HNC_RELEASE_ID/assets?name=kubectl-hns'
 # Build Docker image
 - name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/hnc/controller:$_HNC_IMG_TAG', 'multi-tenancy/incubator/hnc']
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$_HNC_IMG_NAME:$_HNC_IMG_TAG', 'multi-tenancy/incubator/hnc']
```

Comment:
Question: the `HNC_IMG_NAME` defaults to `hnc-manager` which is different from `hnc/controller` here?

Reply:
Yes - we switched the makefile from `hnc/controller` to `hnc-manager` some time ago. Having multiple paths in a Docker registry isn't standard - the Docker Hub, for example, doesn't support it.

Reply:
Thanks for the explanation!